ios/test_app.rb

```diff
@@ -38,6 +38,18 @@ def bundle_identifier(project_root, target_platform)
   @test_app_bundle_identifier
 end
 
+def react_native_path_from_manifest(project_root, target_platform) 
+  manifest_path = find_file('app.json', project_root)
+  unless manifest_path.nil?
+      manifest = JSON.parse(File.read(manifest_path))
+      platform_config = manifest[target_platform]
+      if !platform_config.nil? && !platform_config.empty?
+        reactNativePath = platform_config['reactNativePath']
+        return reactNativePath if reactNativePath.is_a? String
```

Comment:
We need to resolve `reactNativePath` to a path relative to where `Podfile` lives.

Reply:
I added the path resolution using `Pathname.new(resolve_module(reactNativePath)) if reactNativePath.is_a? String`, would it not suffice?

Reply:
Can you post what `Podfile.lock` looks like after running `pod install`? We need to make sure we don't get absolute paths in there.

Reply:
Pushed the podfile.lock.